data/110/873/260/3/1108732603.geojson

```diff
@@ -5,16 +5,16 @@
     "edtf:cessation":"uuuu",
     "edtf:inception":"uuuu",
     "geom:area":0.185123,
-    "geom:area_square_m":1622352379.128015,
+    "geom:area_square_m":1622352557.485719,
     "geom:bbox":"37.986778,44.581104,38.591133,45.190545",
     "geom:latitude":44.867684,
     "geom:longitude":38.283528,
     "iso:country":"RU",
     "label:eng_x_preferred_longname":[
-        "Abinskiy District"
+        "Abinskiy Raion"
```

Comment:
This looks very similar to the `rus_latn` longname we have on this record, though the spelling is slightly different. Expected?

Reply:
Ohhhh, those `rus_latn` longnames are useful!

Reply:
Reset back to `District` via https://github.com/whosonfirst-data/whosonfirst-data-admin-ru/pull/24/commits/66601ef140b14b95d1c32a7684b487fb2feee829.

(Someday we can dive into all the different types of districts and add those as alt long names...)

Reply:
looks good!